src/main/java/org/jabref/logic/layout/format/HTMLChars.java

```diff
@@ -173,6 +218,28 @@ public String format(String inField) {
                 incommand = false;
                 escaped = false;
             }
+        } if (!(field.length() < 0)) {
```

Comment:
Should this be `field.length() == 0` (maybe I misunderstood something)? And maybe the `else` is not needed for this check, since entering the for-loop will already be counted!

Reply:
Yes you are right, will change it.